scroll-animations-1/Overview.bs

```diff
@@ -835,12 +835,20 @@ spec: cssom-view-1; type: dfn;
 	as step 1 of [=update animations and send events=].
 
 	During step 7.14.1 of the [HTML Processing Model](https://html.spec.whatwg.org/multipage/webappapis.html#processing-model-8),
-	any created [=scroll progress timelines=] or [=view progress timelines=] are collected into an <dfn export>initially stale</dfn> set.
-	After step 7.14.1 if there are any [=initially stale=] timelines, they update their current time,
-	the set of [=initially stale=] timelines is cleared and
+	any created [=scroll progress timelines=] or [=view progress timelines=] are collected into a <dfn export>stale timelines</dfn> set.
+	After step 7.14 if any timelines' [=named timeline ranges=] have changed,
+	these timelines are added to the [=stale timelines=] set.
+	If there are any [=stale timelines=], they now update their current time and associated ranges,
+	the set of [=stale timelines=] is cleared and
 	we run and we run an additional step to recalculate styles and update layout.
-	As we only gather stale timelines during the first style and layout calculation,
+
+	Note: We check for layout changes after dispatching any {{ResizeObserver}}s intentionally
```

Comment:
... but this means that ResizeObserver sees an irrelevant layout state vs. what we actually intend to render?

Reply:
That's correct. It's a tradeoff I think. Either:
* Run before ResizeObserver, then any changes to sizes of content made by ResizeObserver may not be handled by scroll driven animations, but ResizeObserver will see layout driven animation results.
* Run after ResizeObserver, then layout animations may not be fully up to date when calling ResizeObserver, but view timelines will be able to observe resizeobserver derived sizes.

Given that ResizeObservers are designed for establishing the size of elements, I feel like view timelines should be updated to reflect the final resize observer output. The converse, animating layout firing ResizeObservers in response to scroll seems like an anti-pattern.

Reply:
We're going to want to use the same mechanism and invalidation for anchor positioning. Chatting with @bfgeek we thought after ResizeObserver would be more in line with the use cases.